pkgs/tools/package-management/rpm/default.nix

```diff
@@ -18,7 +18,8 @@ stdenv.mkDerivation rec {
   buildInputs = [ cpio zlib bzip2 file libarchive nspr nss db xz python lua ];
 
   # rpm/rpmlib.h includes popt.h, and then the pkg-config file mentions these as linkage requirements
-  propagatedBuildInputs = [ popt elfutils nss db bzip2 libarchive libbfd ];
+  propagatedBuildInputs = [ popt nss db bzip2 libarchive libbfd ]
+    ++ stdenv.lib.optionals stdenv.isLinux elfutils;
```

Comment:
The checker marks line 21.  `optionals` takes a _list_ as the second parameter, so either use a singleton list or `optional` instead.